modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayButton from '@clayui/button';
+import ClayCard from '@clayui/card';
+import {useResource} from '@clayui/data-provider';
+import ClayDropDown from '@clayui/drop-down';
+import ClayForm, {ClayInput, ClayRadio, ClayRadioGroup} from '@clayui/form';
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+
+const AdminTooltip = ({
+	namespace,
+	setRemovePinHandler,
+	setShowTooltip,
+	showTooltip,
+}) => {
+	const [pinLabel, setPinLabel] = useState(showTooltip.details.label);
+	const [linkedValue, setLinkedValue] = useState(
+		showTooltip.details.linked_to_sku
+	);
+	const [sku, setSku] = useState(showTooltip.details.sku);
+	const [quantity, setQuantity] = useState(showTooltip.details.quantity);
+	const [networkStatus, setNetworkStatus] = useState(4);
+	const initialLoading = networkStatus === 1;
+	const loadingAd = networkStatus < 4;
+	const error = networkStatus === 5;
+
```

Comment:
Separate these variable declarations into two groups. one that uses `useState` the other that doesn't.
Basically put an empty line on line 34